src/index.js

```diff
@@ -7,6 +7,8 @@ const { pushTask } = require('./modules/pushTask.js');
 const list = new Lists();
 
 const form = document.querySelector('.input');
+const message = document.querySelector('.message');
+message.classList.toggle('on');
 form.addEventListener('submit', (event) => {
   event.preventDefault();
   const name = document.querySelector('#name').value;
```

Comment:
[Optional] You have a great design implemented. I suggest that you trim the entered text before saving them in order to avoid a user saving text comprised of only spaces as their name as shown below:

![image](https://user-images.githubusercontent.com/20554583/202489467-598540ab-ccd8-4be6-8dbc-d5332f41c5b1.png)

Reply:
Okay thank you for pointing that out @emmiiorji , and thank you for your feedback, I will have that fixed.